core/renderers/block_rendering_rewrite/block_render_draw.js

```diff
@@ -198,11 +208,13 @@ Blockly.blockRendering.Drawer.prototype.drawBottom_ = function() {
   for (var i = elems.length - 1; i >= 0; i--) {
     var elem = elems[i];
     if (elem.type === 'next connection') {
-      this.steps_.push(Blockly.blockRendering.constants.NOTCH_PATH_RIGHT);
+      //this.steps_.push('h',  (this.RTL ? 0.5 : - 0.5));
```

Comment:
Do these commented out lines need to be removed?

Reply:
As discussed, these are going to be cleaned up in the next PR as part of additional work.